Tests/TurfTests/JSONTests.swift

```diff
@@ -8,6 +8,10 @@ class JSONTests: XCTestCase {
         XCTAssertEqual(JSONValue(rawValue: 3.1415 as NSNumber), .number(3.1415))
         XCTAssertEqual(JSONValue(rawValue: false as NSNumber), .boolean(false))
         XCTAssertEqual(JSONValue(rawValue: true as NSNumber), .boolean(true))
+        XCTAssertEqual(JSONValue(rawValue: false), .boolean(false))
+        XCTAssertEqual(JSONValue(rawValue: true), .boolean(true))
```

Comment:
Not sure whether supporting this was intentional @1ec5, but the old implementation did allow it, so I kept the behavior and added tests to verify it.